pkg/daemon/daemon.go

```diff
@@ -554,6 +613,245 @@ func (dn *Daemon) detectEarlySSHAccessesFromBoot() error {
 	return nil
 }
 
+// RunHypershift is the entry point for the simplified Hypershift mode daemon
+func (dn *Daemon) RunHypershift(stopCh <-chan struct{}, exitCh <-chan error) error {
```

Comment:
I think it would make sense to move all of the hypershift specific funcs to a seperate file and out of daemon.go

Reply:
I think eventually we would like to de-dupe a lot of this (especially when we eventually move towards layering), so I just kept the code paths in here for now. All the functions should indicate that it is Hypershift-only